src/query_parser/v2/lexer.rl

```diff
@@ -119,6 +120,19 @@ main := |*
       fbreak;
     }
   };
+  vector_range => {
+    tok.pos = ts-q->raw;
+    tok.len = te - ts;
+    tok.s = ts;
+    if (StopWordList_Contains(q->opts->stopwords, "vector_range", strlen("vector_range"))) {
+      RSQuery_Parse_v2(pParser, VECTOR_RANGE_S, tok, q);
+    } else {
+      RSQuery_Parse_v2(pParser, VECTOR_RANGE_T, tok, q);
```

Comment:
Avoid checking for stopword and treats the same way.

Reply:
Why? Stop-words and terms are different tokens. If we want to ensure a proper fallback of `vector_range`, this is the way to do so...

Reply:
to be decided. Currently, we have such a mechanism for `AS` keyword, which will be replaced by the `yield` attribute.